lib/shopify-cli/project.rb

```diff
@@ -80,6 +80,10 @@ def write(ctx, app_type, identifiers = {})
         ctx.write('.shopify-cli.yml', YAML.dump(content))
       end
 
+      def project_name
```

Comment:
Should this be `project_name`? It feels weird to me to have Project.project_name. project project. Although I guess Project.name would be too generic. 🤷

Reply:
We also have `current_project_type` which could be renamed to `current_type`.